benches/src/PoseidonT3_asm.sol

```diff
@@ -0,0 +1,391 @@
+/// SPDX-License-Identifier: MIT
+pragma solidity >=0.7.0;
+
+library PoseidonT3 {
```

Comment:
I believe you should mention the author/repository of this Solidity implementation.